can-bus/dbcs/CanMsgs.dbc

```diff
@@ -0,0 +1,921 @@
+VERSION ""
+NS_:
+BS_:
+
+
+BU_: DIM BMS DCM GSM INVL INVR FSM PDM DEBUG
+
+
+BO_ 502 DIM_Vitals: 1 DIM
+SG_ Heartbeat : 0|1@1+ (1,0) [0|1] "" DEBUG,BMS
+SG_ State : 1|1@1+ (1,0) [0|0] "" DEBUG,BMS
+
+BO_ 503 DIM_Switches: 1 DIM
+SG_ StartSwitch : 0|1@1+ (1,0) [0|1] "" DEBUG
+SG_ AuxSwitch : 1|1@1+ (1,0) [0|1] "" DEBUG
+
```

Comment:
we don't track dbc on git anymore, this file can be deleted

Reply:
done